resources/log4j.properties

```diff
@@ -20,5 +20,6 @@ log4j.appender.metabase=metabase.logger.Appender
 log4j.logger.metabase=INFO
 log4j.logger.metabase.db=DEBUG
 log4j.logger.metabase.driver=DEBUG
+log4j.logger.metabase.middleware=DEBUG
```

Comment:
just to make sure we are clear.  i am fine with including errors and failures in the logging output, but i don't believe it makes sense to print out everything that happens in the app.  if something is an actual error then it should be logged at the appropriate log level such as WARN/ERROR/FATAL

the primary reason we have a DEBUG level in our logger is so that we can log messages at that level which are only printed specifically when they are needed.  those are things that aren't necessary or useful during normal execution of the app.